pkg/controllers/nodeclass/suite_test.go

```diff
@@ -889,6 +890,118 @@ var _ = Describe("NodeClassController", func() {
 			nodeClass = ExpectExists(ctx, env.Client, nodeClass)
 			Expect(nodeClass.Annotations[v1beta1.AnnotationEC2NodeClassHash]).To(Equal(expectedHash))
 		})
+		It("should update ec2nodeclass hash version when the ec2nodeclass hash version is out of sync with the controller", func() {
```

Comment:
```suggestion
		It("should update ec2nodeclass hash version when the ec2nodeclass hash version is out of sync with the controller hash version", func() {
```

Same comments as in the other repo, we should be really explicit about what we are comparing and is saying is "out of sync" here